code/modules/shuttle/shuttle.dm

```diff
@@ -640,6 +640,13 @@
 					var/obj/machinery/atmospherics/supermatter_crystal/bakoom = AM
 					addtimer(CALLBACK(bakoom, TYPE_PROC_REF(/obj/machinery/atmospherics/supermatter_crystal, explode), bakoom.combined_gas, bakoom.power, bakoom.gasmix_power_ratio), 1 SECONDS)
 				continue
+			// Your mech will not save you.
+			if(ismecha(AM))
+				var/obj/mecha/mech = AM
+				if(mech.occupant)
+					var/mob/pilot = mech.occupant
+					mech.Destroy()
+					pilot.gib()
```

Comment:
You'll need these to be wrapped in an INVOKE_ASYNC call to make this pass CL

Reply:
Also, never call Destroy directly. Always use `qdel`